SECURITY.md

```diff
@@ -0,0 +1,23 @@
+<!-- BEGIN TYRO SECURITY.MD V0.0.4 BLOCK -->
+
+# Security
+
+## Reporting Security Issues
+
+If you find a security issue, or evidence of one, please report it privately using [Github's privately reporting a vulnerability feature](https://docs.github.com/en/code-security/security-advisories/guidance-on-reporting-and-writing/privately-reporting-a-security-vulnerability#privately-reporting-a-security-vulnerability).
+
+[Report a vulnerability](https://github.com/tyro/<YOU_REPO_HERE>/security)
```

Comment:
should we replace the <YOUR_REPO_HERE> with our repo name?

Reply:
yep thanks, good catch